Formula/rust.rb

```diff
@@ -4,14 +4,14 @@ class Rust < Formula
   license any_of: ["Apache-2.0", "MIT"]
 
   stable do
-    url "https://static.rust-lang.org/dist/rustc-1.64.0-src.tar.gz"
-    sha256 "b3cd9f481e1a2901bf6f3808d30c69cc4ea80d93c4cc4e2ed52258b180381205"
+    url "https://static.rust-lang.org/dist/rustc-1.65.0-src.tar.gz"
+    sha256 "5828bb67f677eabf8c384020582b0ce7af884e1c84389484f7f8d00dd82c0038"
 
     # From https://github.com/rust-lang/rust/tree/#{version}/src/tools
     resource "cargo" do
       url "https://github.com/rust-lang/cargo.git",
-          tag:      "0.65.0",
-          revision: "387270bc7f446d17869c7f208207c73231d6a252"
+          tag:      "0.66.0",
```

Comment:
```suggestion
          # tag:      "0.66.0",
```

Can do this for now. It'll get it based on the commit.

Reply:
Done, thanks.